profile2/impl/src/java/org/sakaiproject/profile2/logic/SakaiProxyImpl.java

```diff
@@ -592,10 +592,10 @@ public MimeTypeByteArray getResource(final String resourceId) {
 				mtba.setMimeType(resource.getContentType());
 				return mtba;
 			} catch (final Exception e) {
-				log.error("SakaiProxy.getResource() failed for resourceId: " + resourceId + " : " + e.getClass() + " : " + e.getMessage());
+				log.warn("SakaiProxy.getResource() failed for resourceId: " + resourceId + " : " + e.getClass() + " : " + e.getMessage());
```

Comment:
Use new logging pattern:
log.warn("Failed to get resource with id: {} : {} : {}", resourceId, e.getClass(), e.getMessage());